### PR TITLE
Adding Teletask Hub, Light and Switch

### PR DESCRIPTION
**Pull request by Tiemooowh**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/teletask.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Teletask"
+description: "Instructions on how to integrate Teletask components with Home Assistant."
+date: 2019-02-06 20:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Hub
```
Comment (klaasnicolaas) [inline source/_components/teletask.markdown]:
```suggestion
ha_category:
  - Hub
  - Light
  - Switch
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Could you include the switch and light page in your component page? We now work with a new format so that we do not create extra pages for the light, switch, sensor etc.

File: source/_components/teletask.markdown (hunk 1 above)
Comment (Tiemooowh) [reply source/_components/teletask.markdown]: merged the pages into one single page

**Comment by Tiemooowh** [comment]
Hi Klaas,

Adjusted the code as per your remarks 👍 

Br
T

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**